.changeset/twelve-seas-battle.md

```diff
@@ -0,0 +1,5 @@
+---
+"@rocket.chat/meteor": patch
+---
+
+Added file id to message exports zip file names in order to avoid overwriting previous exports generated in the same day when using external storage services (such as Amazon S3)
```

Comment:
I think changeset doesn't have to mention things like ids and just get to wat was fixed.
```suggestion
Fixed an issue where old exports would get overwritten by new ones if generated on the same day, when using external storage services (such as Amazon S3)
```

Reply:
Thanks for the suggestion! That's much better